network/transport/grpc/authenticator.go

```diff
@@ -47,10 +48,18 @@ type tlsAuthenticator struct {
 }
 
 func (t tlsAuthenticator) Authenticate(nodeDID did.DID, grpcPeer grpcPeer.Peer, peer transport.Peer) (transport.Peer, error) {
+	withOverride := func(peer transport.Peer, err error) (transport.Peer, error) {
+		if peer.AcceptUnauthenticated {
+			log.Logger().Warnf("Connection manually authenticated, authentication error: %v", err)
+			return peer, nil
+		}
+		return peer, err
+	}
+
```

Comment:
If `peer.AcceptUnauthenticated` is `true` can't we just skip the entire `Authenticate` method altogether? Doesn't really make a difference right?

Reply:
That would also remove the logging. So you wouldn't be aware of a faulty bootstrap node.

Reply:
Right, makes sense!